tests/framework/installer/ceph_helm_installer.go

```diff
@@ -63,10 +68,166 @@ func (h *CephInstaller) CreateRookCephClusterViaHelm(values map[string]interface
 	}
 	values["cephClusterSpec"] = clusterCRD["spec"]
 
+	const blockPoolName = "ceph-block-test"
+	const blockPoolSCName = "ceph-block-test-sc"
+	if err := h.CreateBlockPoolConfiguration(values, blockPoolName, blockPoolSCName); err != nil {
+		return err
+	}
+	const filesystemName = "ceph-filesystem-test"
+	const filesystemSCName = "ceph-filesystem-test-sc"
+	if err := h.CreateFileSystemConfiguration(values, filesystemName, filesystemSCName); err != nil {
+		return err
+	}
+	const objectStoreName = "ceph-objectstore-test"
+	const objectStoreSCName = "ceph-bucket-test-sc"
+	if err := h.CreateObjectStoreConfiguration(values, objectStoreName, objectStoreSCName); err != nil {
+		return err
+	}
+
 	logger.Infof("Creating ceph cluster using Helm with values: %+v", values)
 	if err := h.helmHelper.InstallLocalRookHelmChart(h.settings.Namespace, CephClusterChartName, values); err != nil {
 		return err
 	}
 
+	storageClassList, err := h.k8shelper.Clientset.StorageV1().StorageClasses().List(context.TODO(), v1.ListOptions{})
```

Comment:
Could you factor out the validation and deleting the helm resources into a separate method? I'm thinking it should be called from the end of the `InstallRook()` test method after the other validations are run to start the cluster, toolbox, etc.